src/renderer/shared/core/types/account.ts

```diff
@@ -43,6 +43,8 @@ export type WalletConnectAccount = Omit<BaseAccount, 'cryptoType'> & {
 
 export type Account = BaseAccount | ChainAccount | MultisigAccount | WalletConnectAccount | ShardAccount;
 
+export type RawAccount<T extends Account> = Omit<NoID<T>, 'accountId' | 'walletId' | 'baseId'>;
```

Comment:
I would suggest another name - `DraftAccount`
`Raw` is too abstract and can confuse without context, whether `Draft` clearly states about shape or unfinished form of something

Reply:
Don't forget to add this type into export file - `index.ts`